float32 resolution
uint32 width
uint32 height
geometry_msgs/Pose origin